### PR TITLE
Initial kingman

### PR DESCRIPTION
**Pull request by JereKoskela**
I cleaned up the printfs and checked the recomb_rates vector for NULL from the end of the previous PR.

### CODE REVIEW

File: lib/sim.c [hunk 1]
```diff
@@ -173,11 +173,9 @@ index_to_pixel_coord(int index, unsigned int N, unsigned int *v)
 double
 beta(int n, int k, double u)
 {
-    double ret = exp(gsl_sf_lnchoose(n, k) + (double) k * log(u)
-        + (double)(n - k) * log(1.0 - u));
```
Comment (jeromekelleher) [inline lib/sim.c:177]: Why change this? I think the line-breaking is better as it is.

File: lib/sim.c [hunk 2]
```diff
@@ -618,6 +625,7 @@ sim_add_individual_to_pixel(sim_t *self, unsigned int pixel, individual_t *ind)
     /* insert the id into this pixel */
     node = sim_alloc_avl_set_node(self, id);
     if (avl_insert_node(&self->P[pixel], node) == NULL) {
+        printf("add_individual_to_pixel error");
```
Comment (jeromekelleher) [inline lib/sim.c]: Can we clear out these printfs, or are they still useful?

File: lib/sim.c [hunk 3]
```diff
@@ -725,6 +733,7 @@ sim_initialise(sim_t *self)
                 goto out;
             }
             if (avl_insert_node(&ind->ancestry, node) == NULL) {
+                printf("Initialise error");
```
Comment (jeromekelleher) [inline lib/sim.c]: Ditto here.

**Comment by jeromekelleher** [comment]
Looks great @JereKoskela, thanks. I've a couple of trivial questions above, but I'm happy to merge after that.

**Comment by JereKoskela** [comment]
Thanks @jeromekelleher. I fixed the three points you raised, and a few other remaining legacy prints I found in sim.c.

**Comment by jeromekelleher** [comment]
Superb; thanks @JereKoskela!

**Pull request merged**